src/app/utils/tests/TestUtils.spec.ts

```diff
@@ -354,19 +360,13 @@ export class ComponentTestUtils<T extends AbstractGameComponent, P extends Compa
     public async expectClickSuccess(elementName: string): Promise<void> {
         return this.expectClickSuccessWithAsymmetricNaming(elementName, elementName);
     }
-    public async expectInterfaceClickSuccess(elementName: string, waitOneMs: boolean = false): Promise<void> {
+    public async expectInterfaceClickSuccess(elementName: string, waitInMs?: number): Promise<void> {
         const context: string = 'expectInterfaceClickSuccess(' + elementName + ')';
-        await this.clickElement(elementName, false, waitOneMs);
+        await this.clickElement(elementName, false, waitInMs);
 
-        expect(this.cancelMoveSpy).not
-            .withContext(context)
-            .toHaveBeenCalledWith();
-        expect(this.chooseMoveSpy).not
-            .withContext(context)
-            .toHaveBeenCalledWith();
-        expect(this.onLegalUserMoveSpy).not
-            .withContext(context)
-            .toHaveBeenCalledWith();
+        expect(this.cancelMoveSpy).not.withContext(context).toHaveBeenCalledWith();
```

Comment:
inverser le withContext et not ? Sinon c'est très bizarre à lire : "j'attends que cancelMoveSpy sans contexte soit appelé"